core/frontend/src/EmphasizeElements.ts

```diff
@@ -234,15 +234,34 @@ export class EmphasizeElements implements FeatureOverrideProvider {
     return true;
   }
 
-  /** Clear elements with color/transparency overrides. Specify key to clear only a single override.
+  /** Clear color/transparency overrides from elements. Removes all overrides when neither key or ids is specified.
+   * @params key Specify key value from [[EmphasizeElements.getOverriddenElements]] or [[EmphasizeElements.createOverrideKey]]
+   * to remove a single color/transparency override for the corresponding elements.
+   * @params ids Specify the IDs of elements to remove any color/transparency override from.
    * @return false if nothing to clear.
    */
-  public clearOverriddenElements(vp: Viewport, key?: number): boolean {
+  public clearOverriddenElements(vp: Viewport, key?: number, ids?: Id64Arg): boolean {

```

Comment:
Can `ids` be `Id64String | Iterable<string>` instead?

Reply:
All the other methods take an Id64Arg...

Reply:
Yeah, we have bad habits. It's a nitpick, ignore if you want.